server/models/test/data.sql

```diff
@@ -198,8 +199,11 @@ INSERT INTO permission (unit_id, user_id) VALUES
 -- Patient Group Module
 (29, 1),
 
+-- Patient documents
+(136, 1),
```

Comment:
In the final implementation, this Patient Documents module will only be available via a link from the Patient Overview page.  However, this is acceptable for now.